attributes/default.rb

```diff
@@ -138,5 +138,7 @@
 # default['openssh']['server']['permit_tunnel'] = 'no'
 # default['openssh']['server']['chroot_directory'] = 'none'
 # default['openssh']['server']['banner'] = 'none'
-# default['openssh']['server']['subsystem'] = 'sftp /usr/libexec/sftp-server'
+
+# default['openssh']['server']['subsystem'] = {sftp: '/usr/lib/openssh/sftp-server' }
+default['openssh']['server']['subsystem'] = {}
```

Comment:
to keep backwards compatibility it should use the default specified on the line above that is commented out.